resources/worldconfig.ini

```diff
@@ -61,3 +61,6 @@ hardcore_lose_uscore_on_death_percent=10
 
 # Allow civilian players the ability to turn the nameplate above their head off.  Must be exactly 1 to be enabled for civilians.
 allow_nameplate_off=0
+
+# Turn logging of IP addresses for anti-cheat reporting on (1) or off(0)
+log_ip_addresses_for_anti_cheat=1
```

Comment:
Will the cmake pick this up and add it to the users existing configs?

Reply:
#1154